osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/security/ReplaceKeyCommandExecutor.java

```diff
@@ -55,7 +55,9 @@
 
   @Autowired private SecretManagementService secretManagementService;
 
-  @Autowired EncryptionHelperService encryptionService;
+  @Autowired
+  @Qualifier("decrypterForGxfSmartMetering")
+  private RsaEncrypter rsaEncrypter;
```

Comment:
Wouldn't it help being able to understand the code a little better if the field `rsaEncrypter` was named like the qualifier (`decrypterForGxfSmartMetering`)?

Reply:
Yes, I changed it